ruby-lsp-rails.gemspec

```diff
@@ -22,6 +22,6 @@ Gem::Specification.new do |spec|
   end
 
   spec.add_dependency("rails", ">= 6.0")
-  spec.add_dependency("ruby-lsp", ">= 0.9.1", "< 0.11.0")
+  spec.add_dependency("ruby-lsp", ">= 0.10.0", "< 0.11.0")
```

Comment:
Not a blocker, but I think it's essentially `~> 0.10.0`?

Reply:
It is, but for some reason dependabot always changes the requirement automatically to use `>=` and `<`.